openmc/deplete/results.py

```diff
@@ -444,9 +435,9 @@ def save(op, x, op_results, t, power, step_ind, proc_time=None):
 
         for i in range(stages):
             for mat_i in range(n_mat):
-                results[i, mat_i, :] = x[offset + i][mat_i][:]
+                results[i, mat_i, :] = x[i][mat_i]
 
-        results.k = [r.k for r in op_results]
+        results.k = [(r.k.nominal_value, r.k.std_dev) for r in op_results]
```

Comment:
Why this change?

Reply:
Commit 98957fb47350d8b012236f00cc89a030c38e4590 in this PR changed how the operator stores the k values. Previously, this was a tuple `[k, unc]`. However, the SIE methods update `k` through the iterations, causing issues attempting to multiply / add tuples of floats.

Converting to `ufloats` allowed the k update to be unchanged, but broke the results saving. This specific change was added in that commit to allow the `Results` object to be able to pass the `OperatorResult.k` `ufloat` into what `h5py` can save during:
https://github.com/openmc-dev/openmc/blob/f633e848226d6eb8f3dad1677c6c2afd1a7a5e04/openmc/deplete/results.py#L325-L326